.github/actions/upload-sentry-sourcemaps/action.yml

```
@@ -1,78 +1,39 @@
-name: "Upload Sentry Sourcemaps"
-description: "Extract sourcemaps from Docker image and upload to Sentry"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)